api/envoy/api/v2/cluster/outlier_detection.proto

```diff
@@ -78,4 +78,21 @@ message OutlierDetection {
   // used to disable ejection or to ramp it up slowly. Defaults to 0.
   google.protobuf.UInt32Value enforcing_consecutive_gateway_failure = 11
       [(validate.rules).uint32.lte = 100];
+
+  // The number of consecutive locally originated failures before ejection
+  // occurs. Defaults to 5.
+  google.protobuf.UInt32Value consecutive_local_origin_failure = 12;
+
+  // The % chance that a host will be actually ejected when an outlier status
+  // is detected through consecutive locally originated failures. This setting can be
+  // used to disable ejection or to ramp it up slowly. Defaults to 100.
+  google.protobuf.UInt32Value enforcing_consecutive_local_origin_failure = 13
+      [(validate.rules).uint32.lte = 100];
+
+  // The % chance that a host will be actually ejected when an outlier status
+  // is detected through success rate statistics for locally originated errors.
+  // This setting can be used to
+  // disable ejection or to ramp it up slowly. Defaults to 100.
```

Comment:
nit: fix up the premature new line in the comment and let fix-format do its thing to wrap the comment.

Reply:
Done.